cpp/ps0004/ps0004_soln.cpp

```diff
@@ -4,20 +4,29 @@ using namespace std;
 
 int main()
 {
- int a = 65;
+    // prompting user for number 1/1.5
+    float num;
+    cin >> num;

```

Comment:
- [x] declared a variable to store the user input
- [ ] chose a suitable data type for the variable
- [x] prompted user to enter a number
- [x] used `std::cin` in a right way to get user's input